distribution/docs/src/main/resources/_contents/_catalog-contents/configuring-catalog-contents.adoc

```diff
@@ -44,7 +44,7 @@
 |Set threshold for running marshalling in parallel
 
 |<<catalog_duplicate_validator, Catalog Duplicate Validator>>
-|${ddf-branding-lowercase}.catalog.metacard.duplication.DuplicationValidator
+|org.codice.${ddf-branding-lowercase}.validator.metacard.duplication.DuplicationValidator
```

Comment:
Yes. I haven't started a specific clean up of them yet, but I'm changing them as I find them.